### ISSUE TITLE
Action Intent Links are missing image and alt text props

### ISSUE THREAD

**Issue by ImLunaHey** [bluesky-social/social-app#7885]
### Describe the Feature

there's currently no way to use "Action Intent Links" with anything other than text. it would be nice if we could have sites share with images + alt text as well.
